suite-native/module-device-settings/src/navigation/DeviceSettingsStackNavigator.tsx

```diff
@@ -5,17 +5,19 @@ import {
     DeviceStackRoutes,
     stackNavigationOptionsConfig,
 } from '@suite-native/navigation';
+import { FirmwareUpdateInProgressScreen } from '@suite-native/firmware';
 
 import { DeviceSettingsModalScreen } from '../screens/DeviceSettingsModalScreen';
 import { DeviceAuthenticityStackNavigator } from './DeviceAuthenticityStackNavigator';
 import { DevicePinProtectionStackNavigator } from './DevicePinProtectionStackNavigator';
-
+import { FirmwareUpdateScreen } from '../screens/FirmwareUpdateScreen/FirmwareUpdateScreen';
+import { ContinueOnTrezorScreen } from '../screens/ContinueOnTrezorScreen';
 const DeviceSettingsStack = createNativeStackNavigator<DeviceSettingsStackParamList>();
 
 export const DeviceSettingsStackNavigator = () => (
     <DeviceSettingsStack.Navigator
         initialRouteName={DeviceStackRoutes.DeviceSettings}
-        screenOptions={stackNavigationOptionsConfig}
+        screenOptions={{ ...stackNavigationOptionsConfig, animation: 'slide_from_bottom' }}
```

Comment:
Not sure the animation is right. It affects also enable/change/disable PIN flows which were already approved by @shenkys.

Reply:
you are right, removing it

Reply:
[61b3b75](https://github.com/trezor/trezor-suite/pull/15184/commits/61b3b75ec0d7f238f34d1faada1ec2054303a0a6)